### PR TITLE
Add Authentication form

### PR DESCRIPTION
**Pull request by okucheruk**
This PR  add Page with AuthenticationForm.

**Highlights**:
- login and password inputs 
- sign in button
- link to another Page

### CODE REVIEW

**Comment by okucheruk** [comment]
@dkalinin, PTAL.

File: app/FileStorageUI.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ export default class FileStorageUI {
    * @param {HTMLElement} parentElement The root element or node for components of this element.
    */
   _render(parentElement) {
-    new SignUpForm(parentElement);
+    // new SignUpForm(parentElement);
```
Comment (dkalinin) [inline app/FileStorageUI.js]: Please remove commented lines.

File: app/view/auth/AuthenticationForm.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        .withTextContent('Create an account.');
+
+    new Button(parentElement)
+        .withClassName('mui-btn')
+        .withTextContent('Sign in')
+        .subscribeToEvent('click', (event) => this._signinButtonClickHandler(event));
+  }
+
+  /**
+   * Handles 'click' event type of signin {@link Button} UI component.
```
Comment (dkalinin) [inline app/view/auth/AuthenticationForm.js]: There's no word 'signin' in English.

File: app/view/auth/AuthenticationForm.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        .withInputId('login_field');
+
+    new InputField(parentElement)
+        .withClassName('form-row')
+        .withLabelText('Password')
+        .withInputId('password_field')
+        .withInputType('password');
+
+    new Anchor(parentElement)
+        .withHref('#/singup')
```
Comment (dkalinin) [inline app/view/auth/AuthenticationForm.js]: singup -> s-i-g-n-u-p

File: app/FileStorageUI.js (hunk 1 above)
Comment (okucheruk) [reply app/FileStorageUI.js]: Removed.

File: app/view/auth/AuthenticationForm.js (hunk 2 above)
Comment (okucheruk) [reply app/view/auth/AuthenticationForm.js]: Fixed.

**Comment by okucheruk** [comment]
Router feature will be available in next pull request. I'm going to merge develop branch into ferature/router and refactor code style today.

**Comment by okucheruk** [comment]
@dkalinin, fixed, PTAL.

**Pull request merged**